University of Leicester/Leicester210.xml

```diff
@@ -0,0 +1,326 @@
+</seg>
+</handNote>
+   </handDesc>
+<decoDesc>
+<decoNote type="ornamentation" xml:id="d1">
+<locus target="#2r"></locus>
+<desc>Ornamental band (incipit page of text 1): coloured (red, yellow, green, and black), with one central and two lateral pendants reaching the bottom of the page; geometric and interlaces motifs.</desc>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d2">
+<locus target="#27r"></locus>
+<desc>Ornamental band (incipit page of text 2): coloured (red, yellow, green, and black), geometric and interlaces motifs.</desc>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d3">
+<locus target="#53rb"></locus>
+<desc>Ornamental band (incipit of text 3): coloured (red, yellow, green, and black), geometric and interlaces motifs.</desc>
+</decoNote>
+</decoDesc>
+<additions>
+<list>
+<item xml:id="a1">
+<desc type="Record">Amharic note describing the content of the manuscript.</desc>
+<note>Black ink; recent hand</note>
+<locus target="#1r"></locus>
+<q xml:lang="am">የቂርቆስ፡ ገድል፡ የገብረ፡ ክርስቶስ፡ …</q>
+</item>
+<item xml:id="a2">
+<desc type="Unclear">Amharic unknown note.</desc>
+<note>Black ink; recent hand (probably same as Additio 1); written upside-down and at the bottom left part of the folio</note>
+<locus target="#1v"></locus>
+</item>
+<item xml:id="a3">
+<desc type="Record">Discourse of the wisdom on the 12 Apostles.</desc>
+<note>Black ink (red ink only for one punctuation); recent hand; not completely readable at the bottom of the folio (affected by humidity).</note>
+<locus target="#26vab"></locus>
+<q xml:lang="gez">ነገረ፡ ጥበብ፡ በእንተ፡ ፲ወ፪ሐወርያተ፡ እም፡ ፲ወ፪ነገደ፡ እስራኤል፡...</q>
+</item>
+<item xml:id="e1">
+<desc type="ScribalSupplication">Supplication formulas (within text 4) with mention of the commissioner's name በኵረ፡ ጽዮን፡ Bakʷǝra Ṣǝyon.</desc>
+<locus target="#70vb"></locus>, <locus target="#72rb"></locus>
+</item>
+<item xml:id="e2">
+<desc type="ScribalSupplication">Supplication formula (within text 4) with mention of the commissioner's name በኵረ፡ ጽዮን፡ Bakʷǝra Ṣǝyon.</desc>
+<locus target="#72rb"></locus>
+</item>
+<item xml:id="e3">
+<desc type="Correction">Scribal corrections written over erasures, e.g. <locus target="#7ra"></locus>, <locus target="#25rb"></locus>, <locus target="#26rb"></locus>.</desc>
+</item>
+<item xml:id="e4">
+<desc type="findingAid">Cotton red threads are inserted in the right margin of <locus target="#53"></locus> for navigating the text.
```

Comment:
These are encoded as part of binding, see https://betamasaheft.eu/Guidelines/?id=bindingDescription

Reply:
This is still unchanged?

Reply:
Thanks Zhenia, Somehow I did not read this correction. And I have just realized that I forgot to add in the description the binding. Now I did it.